composer.json

```diff
@@ -69,7 +70,7 @@
         "coverage": "phpunit --coverage-html=.phpunit.cache/code-coverage",
         "cs": "php-cs-fixer fix --verbose --dry-run --diff",
         "fix": "php-cs-fixer fix --verbose --diff",
-        "phpstan": "phpstan analyze --memory-limit 512M",
```

Comment:
Without this, I get the following when running `composer phpstan` locally:

```
     Child process error: PHPStan process crashed because it reached configured PHP memory limit: 128M  
     Increase your memory limit in php.ini or run PHPStan with --memory-limit CLI option.               
      while running parallel worker                                                                     
```

Reply:
Was it with PHPStan 2?
We can just add this parameter again, but my experience has been that memory requirements have decreased with this new version, and I have not experienced a need for it in SimplePie or FreshRSS.
But some OS seem to be less efficient though. Are you maybe running on something else than Linux?

Reply:
Yes, on this branch after running `composer install`. I am on Linux. Maybe your environment has higher than the default memory limit? https://github.com/php/php-src/blob/e6c07b004b654dcc2362bd0ea29c269e4a531a5a/php.ini-production#L430